basis/graph/configured_node.py

```diff
@@ -67,13 +67,13 @@ class NodeInterface(FrozenPydanticBase):
 class ConfiguredNode(FrozenPydanticBase):
     name: str
     node_type: NodeType
-    absolute_node_path: NodePath
+    id: NodeId
     # declared ports
     interface: NodeInterface
     # distance from root graph, starts at 0
     node_depth: int
     description: str = None
-    parent_node: NodePath = None
+    parent_node_id: NodeId = None
     file_path_to_node_script_relative_to_root: str = None
     # Configuration
     parameter_values: Dict[str, Any]
```

Comment:
This actually applies to line 103 (no way to comment on unchanged lines in gh...?), get_node will no longer work. Ideally have both get node by id and a find nodes by name methods, the latter finding all nodes with the matching name?

Reply:
good catch. Added tests to those functions.